docs/releases/index.md

```diff
@@ -39,11 +39,11 @@ Release highlights:
      3PAR OS 3.3.1, 3.3.2
    </td>
  </tr>
- <!-- tr>
+ <tr>
    <th>Blogs</th>
    <td>
-    <a href="https://FIXME">HPE Alletra Observability for DevOps and Cloud Native with Prometheus</a><br />
-    <a href="https://FIXME">Get started with Prometheus and HPE Alletra</a> (tutorial) <br />
+    Release blog on <a href="https://community.hpe.com/t5/Around-the-Storage-Block/HPE-CSI-Driver-for-Kubernetes-enhancements-with-monitoring-and/ba-p/7158137">Around The Storage Block</a><br />
```

Comment:
I would have expected the "what", not the "where", to be hyperlinked.  For example:

```
    <a href="https://community.hpe.com/t5/Around-the-Storage-Block/HPE-CSI-Driver-for-Kubernetes-enhancements-with-monitoring-and/ba-p/7158137">Release blog</a> on Around The Storage Block<br />
    <a href="https://developer.hpe.com/blog/get-started-with-prometheus-and-grafana-on-docker-with-hpe-storage-array-exporter/">Tutorial using Docker</a> on HPE Developer Community
```

Maybe I'm unfamiliar with a convention.

Reply:
It can go either way. I prefer it this way because linking to ATSB would assume linking to the blog itself, not the actual post. The blog names won't fit in the table.

Reply:
Ok, I see what I did now, I'm fixing it. :/